stor/tests/test_dx.py

```diff
@@ -1005,6 +1005,8 @@ def _test_proxy_url(self, proxy_path):
                 with pytest.raises(ValueError, match='filename MUST match object name'):
                     dx_p.temp_url(filename='another_name.txt')
 
+            assert proxy_path in dx_p.temp_url()
```

Comment:
Needs something where it doesn’t have trailing slash on it?

Reply:
it already has such example:

```
self._test_proxy_url("https://myproxy.example.com/dnax-gateway")
```

down below in `test_on_file_with_proxy_url`